package.json

```diff
@@ -17,10 +17,10 @@
   "license": "MIT License",
   "scripts": {
     "prepare": "is-ci || husky install",
-    "dev": "ENV_PATH=.env.development node ace serve --watch",
+    "dev": "cross-env-shell ENV_PATH=.env.development node ace serve --watch",
```

Comment:
So this means no one ever used the ferdium-server for development on a Windows machine before? :thinking:

Reply:
I usually did a docker container to test it out... but this time I took the time to see what was failing xD